openmrs/apps/registration/app.json

```diff
@@ -103,15 +103,11 @@
             },
             "patientSearch": {
                 "address": {
-                    "label": "Rural Ward",
-                    "placeholder": "Enter ward",
-                    "field": "address2"
+                     "label": "Tehsil",
```

Comment:
What is a 'Tehsil' in the context of POC?

Reply:
Tehsil is not related to POC.
It's added for completing the functionality once we get the actual requirement of Address information levels then we will upload Mozambique address information.
After that, we will update the 'Tehsil' to the relevant Address.

Reply:
I think for now lets leave it as rural Ward. We can change it once we get the requirements